content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -588,3 +588,265 @@ As part of the Peruvian localization, besides creating credit notes from an exis
+- While the Sender’s Delivery Guide is a document that justifies the shipment of products 
+(and issued by the sender), the Carrier Delivery Guide justifies the transportation service performed by the driver (or carrier). 
+- Learn more `here <https://tefacturo.pe/blog/sunat/guia-de-remision-electronica/guia-de-remision-transportista/>`_. 
+- *Not included in Odoo*.
+
+Transportation Types
+********************
+
+Private
+^^^^^^^
+
+This option is used when the transfer of **goods is carried out by the owner themself** with 
+their own vehicles. For this case, a Sender’s Delivery Guide must be issued. 
+
+Public
+^^^^^^
+
+This option is used when the **goods are moved by an external carrier**. In this case, 
+two Delivery Guides must be issued; the Sender’s Delivery Guide and the Carrier’s Delivery Guide.
+
+Changes in Version 2.0
+**********************
+
+Direct Submission to SUNAT
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The creation of the GRE (Delivery Guide) in Odoo **must be sent directly to the SUNAT** regardless 
+of the electronic document provider: IAP, Digiflow or SUNAT.
+
+New Required Information
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Version 2.0 requires additional information in the general configuration as well as vehicles, contacts, 
+and products. In the general configurations, it will be necessary to add new credentials that 
+are acquired in the SUNAT portal.
+
+Flow
+****
+
+.. image:: peru/peru_flow_remision.png
+   :align: center 
+   :alt: Flow for Guia de Remision emission
+   
+Cancellations
+^^^^^^^^^^^^^
+
+Currently, the SUNAT no longer uses the term *“Anula”*, but now uses the term *“Dar de baja”*. 
+Therefore, **both** the sender and the carrier can cancel the electronic waybill, *so long as 
+the following conditions are met*: 
+
+- If the shipment has not yet been initiated
+- If the shipment has been initiated, but the receiver must be changed before reaching 
+the final destination.
+
+Testing
+^^^^^^^
+
+The SUNAT did not release any test environment; **this means that any Delivery Guides that 
+were generated by mistake will be sent to the SUNAT**.
+
+If by mistake the Waybill was created in this environment, it will be necessary to be deleted 
+from the SUNAT portal.
+
+Odoo Configuration
+******************
+
+.. important::
+   - Electronic Sender’s Waybill is currently the only supported type of Waybill in Odoo.
+   - The delivery guide is dependant on the **Inventory App**, the `l10n_pe_edi` and 
+`l10n_pe` modules.
+   
+After following the steps to configure the :doc:`Electronic invoicing <accounting_settings>` 
+and the :doc:`Master data <configura_master_data>`, go to **Apps**, **clear** any default 
+filters set, search for the `l10n_pe_edi_stock_20` module and click the *Activate* button.
+
+Next, you will need to *obtain* the Client ID and Client Secret. To do this, follow the 
+instructions in this `link <https://cpe.sunat.gob.pe/sites/default/files/inline-files/Manual_Servicios_GRE.pdf>`_.
```

Comment:
```suggestion
After following the steps to configure the :ref:`electronic invoicing <peru/accounting-settings>`
and the :ref:`master data <peru/master_data>`, navigate to :menuselection:`Apps` and clear any
default filters set. Search for the `l10n_pe_edi_stock_20` module and click the :guilabel:`Activate`
button.

Next, you will need to obtain the *client ID* and *client secret* from SUNAT. To do so, follow the
`manual de servicios web plataforma nueva GRE
<https://cpe.sunat.gob.pe/sites/default/files/inline-files/Manual_Servicios_GRE.pdf>`_.
```

Reply:
Add `.. _peru/master_data:` and `.. _peru/accounting-settings:` custom anchors above section headings for :ref:s